typed-racket-more/typed/xml.rkt

```diff
@@ -0,0 +1,102 @@
+#lang typed/racket/base
+
+(provide (all-defined-out))
+
+
+(define-type XExpr-Attribute
+  (List Symbol String)
+  )
```

Comment:
Can you fix the `)` on their own lines?

Reply:
sure, I will follow your style, any more formatting fixes?